From fb1f4170d008315cb9eabe994038977a0366ede5 Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Fri, 19 Sep 2014 00:23:42 +0200
Subject: [PATCH] pty: include linux/ioctl.h for TIOCSIG

TIOCSIG is linux specific, so include the linux ioctl header to make sure
it's defined. We currently rely on some rather non-obvious recursive
includes. Make sure its always defined regardless of the system headers.
---
 src/shared/pty.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/pty.c b/src/shared/pty.c
index 2863da489c..31123e5af2 100644
--- a/src/shared/pty.c
+++ b/src/shared/pty.c
@@ -45,6 +45,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <limits.h>
+#include <linux/ioctl.h>
 #include <pty.h>
 #include <signal.h>
 #include <stdbool.h>
